oneview_redfish_toolkit/blueprints/event_service.py

```diff
@@ -16,20 +16,54 @@
 
 import logging
 
-
+from copy import deepcopy
 from flask import abort
 from flask import Blueprint
+from flask import request
 from flask import Response
 from flask_api import status
 
 from oneview_redfish_toolkit.api.errors import OneViewRedfishError
-from oneview_redfish_toolkit.api.event_service \
-    import EventService
+from oneview_redfish_toolkit.api.event import Event
+from oneview_redfish_toolkit.api.event_service import EventService
+
 from oneview_redfish_toolkit import util
 
 
 event_service = Blueprint("event_service", __name__)
 
+ONEVIEW_TEST_ALERT = {
+    "timestamp": "2018-02-12T20:12:03.231Z",
+    "resource": {
+        "category": "alerts",
+        "associatedResource": {
+            "resourceName": "0000A66101, bay 3",
+            "resourceUri": "/rest/server-hardware/"
+                           "30373737-3237-4D32-3230-313530314752"
+        }
+    }
+}
+
+ONEVIEW_TEST_TASK = {
```

Comment:
Maybe use resource in the name instead of task.

Reply:
I am not sure whether resource is the best way to describe this variable. What do you think about using `alert` for SCMB messages containing alerts and use the term `message` for "regular" messages? We will end up having something like ONEVIEW_TEST_ALERT and ONEVIEW_TEST_MESSAGE. Any better idea?